src/utils/animation-frame.service.ts

```diff
@@ -9,9 +9,11 @@ export class AnimationFrameServiceSingleton implements OnDestroy {
 
 	protected animationFrameId: number;
 
-	constructor() {
+	constructor(protected ngZone: NgZone) {
```

Comment:
Maybe this needs to be optional? It's expecting an `ngZone` argument in `utils.module`

Reply:
This is what I get for not running my changes locally 😂